src/api/event.ts

```diff
@@ -27,10 +27,6 @@ export class Event {
   async getAccountEventsByCreationNumber(args: {
     address: string;
     creationNumber: AnyNumber;
-    options?: {
-      pagination?: PaginationArgs;
-      orderBy?: OrderBy<GetEventsResponse[0]>;
-    };
```

Comment:
I been thinking, instead of removing it, should we add pagination for all /api query func? So they are consistent. Any thought?

Reply:
not all queries need pagination. for example, this one.